### DISCUSSION TITLE
Using Regional Prompter with Lora Block Weight extension

### DISCUSSION THREAD

**Discussion by Disguised-Giraffe** [hako-mikan/sd-webui-regional-prompter#231] [General]
In the readme it's mentioned that the Lora Block Weights extension may help the latent mask mode. Are there any presets that are recommended, such as ALL0.5 or INS? Is there a link to discussion about using these two extensions together?

**Comment by hako-mikan** [comment]
This is a challenging issue. Essentially, when you want to use multiple LoRAs, I recommend first reducing the strength of LoRA. The setting of LoRA Block Weight depends on the nature of the LoRA you want to use. If it's a character LoRA, you should probably use MIDD or something similar. A common problem is that the image becomes very coarse due to the influence of OUT9~11 of LoRA.
